tests/Integration/ApiPlatform/EndPoint/ApiAccessTokenEndpointTest.php

```diff
@@ -62,7 +62,12 @@ public function testApiAccessToken(): void
                 'customer_group_read',
             ],
         ]];
-        $options = ['extra' => $parameters];
+        $options = [
```

Comment:
For the access token endpoint, I agree it makes sense to specify the content type But I believe there are multiple ways to specify the values 🤔 For example in my client I was using Multipart form data instead of form URL encoded

The curl request would be equivalent to:
```
curl --request POST \
  --url http://prestashop.api.local/admin-dev/api/oauth2/token \
  --header 'Content-Type: multipart/form-data' \
  --header 'User-Agent: insomnium/0.2.3' \
  --cookie PHPSESSID=6u89otd0qr1njktri5eubunpmh \
  --form grant_type=client_credentials \
  --form client_id=test \
  --form client_secret=d5086ef7ee7f7ff01040bd548a185e48 \
  --form 'scope[]=api_access_read' \
  --form 'scope[]=api_access_write' \
  --form 'scope[]=product_write'
```

So maybe it ould be interesting to add some tests here to check multiple ways of using this endpoint (at least `application/x-www-form-urlencoded` and `multipart/form-data` for now), with a @dataProvider this shouldn't make the test class that much bigger

Reply:
I added a dataprovider